tests/scenario/modules/assets/asset-group.test.ts

```diff
@@ -27,18 +33,6 @@ describe("AssetsGroupsController", () => {
   const sdk = setupHooks();
 
   it("can create a group", async () => {
-    const missingIdQuery: Omit<ApiGroupCreateRequest, "_id"> = {
-      controller: "device-manager/assetsGroup",
-      engineId: "engine-ayse",
-      action: "create",
-      body: {
-        name: "root group",
-      },
-    };
-    await expect(sdk.query(missingIdQuery)).rejects.toThrow(
-      /^Missing argument "_id".$/
-    );
-
     const missingBodyQuery: Omit<ApiGroupCreateRequest, "body"> = {
```

Comment:
what about name missing ??

Reply:
I have added a `BadRequestError` for this case